src/base/bittorrent/session.cpp

```diff
@@ -1124,6 +1124,7 @@ void Session::configure(libtorrent::settings_pack &settingsPack)
             }
         }
 
+        settingsPack.set_str(libt::settings_pack::outgoing_interfaces, networkInterface().toStdString());
```

Comment:
I'm not sure that do understand you. That code sets multiple listening IPs in a loop, while here we want to pass list of interfaces for outgoing connections. Actually, due to our settings UI, this list is either empty or contains a single entry.